README.md

```diff
@@ -241,35 +279,50 @@ So we will create a pull request on GITHUB from main
-7. Tell at your team to pull develop to be up to date. 
+7. Tell at your team to pull develop to be up to date.
+
+## Our application in production
 
-## Our application in production 
-### GITHUB pages 
-Our application is deployed via GITHUB pages. 
+### GITHUB pages
+
+Our application is deployed via GITHUB pages.
 
 ![deployement](./assets/images/img/readme/github-pages.png)
 
-These actions is i your project settings. 
+These actions is i your project settings.
+
 1. Go to Pages section
 2. Chose 'Deploy from a branch', default choice normally
 3. Chose the production branch
-4. Chose the file where there is the index page. 
-5. You can custom your domain, if you to do this please follow this [documentation](https://docs.github.com/fr/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages) but a check of DNS is required so that your app will not instanly deployed. If you don't want a custom domain click just on save. 
+4. Chose the file where there is the index page.
+5. You can custom your domain, if you to do this please follow this [documentation](https://docs.github.com/fr/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages) but a check of DNS is required so that your app will not instanly deployed. If you don't want a custom domain click just on save.
 6. Yous app is deployed :rocket:
 
 ## Our first release
+
 ### First steps
-We began by create a new repository on GITHUB and give access to all team members. Then we chose a bootstrap theme to help us by saving time and to have the possibilituy to be focused on our workflow. 
 
-Then we update the template, each members did a feature, stage their changes, commit them and push to create a PR and ask a review. 
-If the review and checks is ok so we can squash and merge. 
+We began by create a new repository on GITHUB and give access to all team members. Then we chose a bootstrap theme to help us by saving time and to have the possibilituy to be focused on our workflow.
+
+Then we update the template, each members did a feature, stage their changes, commit them and push to create a PR and ask a review.
+If the review and checks is ok so we can squash and merge.
 
 ### Prepare release
-At the end of the first sprint we created our first branch release and realize whole hotfix which are needs. 
+
+At the end of the first sprint we created our first branch release and realize whole hotfix which are needs.
 After that we merge the release branch in our main branch (production branch).
 
-### Deployement 
-After the release step we deployed our main branch from GITHUB pages. 
+### Deployement
+
+After the release step we deployed our main branch from GITHUB pages.
 
 ### Sprint cycle
-![sprint cycle](./assets/images/img/readme/sprint-cycle.png)
\ No newline at end of file
+
+![sprint cycle](./assets/images/img/readme/sprint-cycle.png)
+
+## Problem encountered
```

Comment:
Please add this point after the help to resolve conflict it's in the same tropic as help/faq